debug/map/canvas.html

```diff
@@ -1,48 +1,44 @@
-<!DOCTYPE html>
-<html>
-<head>
-	<title>Leaflet debug page</title>
-
-	<link rel="stylesheet" href="../../dist/leaflet.css" />
-
-	<link rel="stylesheet" href="../css/screen.css" />
-
-	<script src="../../dist/leaflet-src.js"></script>
-</head>
-<body>
-
-	<div id="map" style="width: 600px; height: 600px; border: 1px solid #ccc"></div>
-
-	<script>
-
-		var tiles = new L.GridLayer();
-
-		tiles.createTile = function(coords) {
-			var tile = document.createElement('canvas'),
-			    ctx = tile.getContext('2d');
-
-			tile.width = tile.height = 256;
-
-			ctx.fillStyle = 'white';
-			ctx.fillRect(0, 0, 255, 255);
-
-			ctx.fillStyle = 'black';
-			ctx.fillText('x: ' + coords.x + ', y: ' + coords.y + ', zoom: ' + coords.z, 20, 20);
-
-			ctx.strokeStyle = 'red';
-			ctx.beginPath();
-			ctx.moveTo(0, 0);
-			ctx.lineTo(255, 0);
-			ctx.lineTo(255, 255);
-			ctx.lineTo(0, 255);
-			ctx.closePath();
-			ctx.stroke();
-
-			return tile;
-		}
-
-		var map = new L.Map('map', {center: new L.LatLng(50.5, 30.51), zoom: 15, layers: [tiles]});
-
-	</script>
-</body>
-</html>
+<!doctype html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<title>Leaflet debug page - Canvas</title>
+		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=0" />
+		<link rel="stylesheet" href="../../dist/leaflet.css" />
+		<link rel="stylesheet" href="../css/screen.css" />
+	</head>
+	<body>
+		<div id="map"></div>
+		<script type="module">
+			import {GridLayer, Map, LatLng} from '../../dist/leaflet-src.esm.js';
+
+			const tiles = new GridLayer();
+
+			tiles.createTile = function (coords) {
```

Comment:
Only thinking out loud: Is it better to refactor this and make it extending the GridLayer class?

Reply:
Could be an option, we're doing this in a lot of places so I don't know if it's best practice?